### DISCUSSION TITLE
Compatible with Flood-UI/flood ?

### DISCUSSION THREAD

**Discussion by Rhilip** [jesec/flood#149] [Q&A] [answered]
Now I want to add 3th party support for flood in another repo, However I found the api endpoint between your repo and the original Flood project ([Flood-UI/flood](https://github.com/Flood-UI/flood/)) are different. Such as:

| jesec/flood | Flood-UI/flood |
| ---- | ----- |
| /api/auth/authenticate | /auth/authenticate |
| /api/torrents/add-urls | /api/client/add | 

As seems all in `/api` route groups now and some function change its namespace.

So if there are some method which can help other developer to determine the type of Flood, Like test a route which exist in jesec/flood but not Flood-UI/flood to see the response code ?
Or add back the compatible route with the original Flood ? ( I think it's not a good idea, and I like your route design more than original )

**Comment by jesec** [comment, marked as answer]
GET /api/auth/verify

https://github.com/jesec/flood/blob/63f34be42783759ecee0a28ea456330ee49d40fb/server/routes/api/auth.ts#L240-L249

Before 4.0, the route is `/auth/verify`. 

Note that per current support policy, legacy versions of Flood are not supported.
